analysis_options.yaml

```diff
@@ -1,6 +1,8 @@
 include: package:pedantic/analysis_options.1.8.0.yaml
 
 analyzer:
+  strong-mode:
+    implicit-casts: false
```

Comment:
For reviewing, I recommend looking at the commits separately so that generated file changes don't clog up the diff